lib/View/utils/course_screen.dart

```diff
@@ -0,0 +1,68 @@
+import 'package:flutter/material.dart';
+import 'package:gocast_mobile/View/utils/custom_bottom_nav_bar.dart';
+import 'package:gocast_mobile/View/utils/constants.dart';
+import 'package:gocast_mobile/View/settings_screen.dart';
+
+class CoursesScreen extends StatelessWidget {
```

Comment:
To keep consistency between files, this should have the same name as its file. (Currently the filename is `course_screen` while the class name is `Course*s*Screen`